hbase-protocol-shaded/src/main/protobuf/TooSlowLog.proto

```diff
@@ -42,4 +42,12 @@ message SlowLogPayload {
   optional int32 multi_gets = 12 [default = 0];
   optional int32 multi_mutations = 13 [default = 0];
   optional int32 multi_service_calls = 14 [default = 0];
+  required Type type = 15;
+
+  enum Type {
+    SLOW_LOG = 0;
+    LARGE_LOG = 1;
```

Comment:
nit: Helps to have a code comment about what a SLOW log is and what a LARGE log is..

Reply:
Done